Jenkinsfile

```diff
@@ -2,22 +2,26 @@
 
 node ('docker') {
 
+    def DEFAULT_REPO = 'https://github.com/jenkinsci/blueocean-plugin.git'
     def NOT_TRIGGERED = 'not-triggered'
 
+
     // Allow the pipeline to be built with parameters, defaulting the
     // Blue Ocean branch name to be that of the ATH branch name. If no such branch
     // of Blue Ocean exists, then the ATH will just run against the master branch of
     // Blue Ocean.
     properties([parameters([
             string(name: 'BLUEOCEAN_BRANCH_NAME', defaultValue: "${env.BRANCH_NAME}", description: 'Blue Ocean branch name against which the tests on this ATH branch will run.'),
-            string(name: 'TRIGGERED_BY_BUILD_NUM', defaultValue: NOT_TRIGGERED, description: 'The Blue Ocean build number, if triggered by the building of a Blue Ocean branch. Used to get pre-assembled Jenkins plugins. Use a valid build number, or "latest" to get artifacts from the latest build. Otherwise just use the default value.')
+            string(name: 'TRIGGERED_BY_BUILD_NUM', defaultValue: NOT_TRIGGERED, description: 'The Blue Ocean build number, if triggered by the building of a Blue Ocean branch. Used to get pre-assembled Jenkins plugins. Use a valid build number, or "latest" to get artifacts from the latest build. Otherwise just use the default value.'),
+            string(name: 'REPO', defaultValue: DEFAULT_REPO, description: 'The repo to build. If you want to validate a fork, you can change this.')
     ]), pipelineTriggers([])])
 
     def branchName;
     def buildNumber;
     try {
         branchName = "${BLUEOCEAN_BRANCH_NAME}"
         buildNumber = "${TRIGGERED_BY_BUILD_NUM}"
+        repoUrl = "${REPO}"
```

Comment:
NIT: I know it doesn't matter in terms of correctness, but how about a `def` for `repoUrl` above with the other vars.